src/User/RegistrationForm.jsx

```diff
@@ -65,8 +65,10 @@ const RegistrationForm = () => {
           />
           <Input
             label="Email Address: "
-            type="text"
+            type="email"
             name="email"
+            pattern="^[a-zA-Z0-9\.]+@[a-zA-Z0-9]+\.[\.A-Za-z]{1,10}"
```

Comment:
You don't like the [default pattern](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email) for type=email?  Yours is quite restrictive.

Reply:
We tried it without the custom regex pattern, but the default setting for type=email doesn't require a ".com" domain after the "@email".